package.json

```diff
@@ -6,6 +6,7 @@
     "@babel/runtime": "^7.7.7",
     "@lerna/prompt": "3.18.5",
     "@types/express": "^4.17.2",
```

Comment:
This will clash with https://github.com/gatsbyjs/gatsby/pull/20626/files

Reply:
easy fix tho!